metals/src/main/scala/scala/meta/internal/metals/doctor/ProblemResolver.scala

```diff
@@ -188,11 +178,12 @@ class ProblemResolver(
       mtagsResolver.isSupportedScalaVersion(version)
 
     val scalaVersionProblem = scalaTarget.scalaVersion match {
-      case version if !isSupportedScalaVersion(version) && scalaTarget.isSbt =>
-        if (ScalaVersions.isFutureVersion(version))
-          Some(FutureSbtVersion)
-        else
-          Some(UnsupportedSbtVersion)
```

Comment:
I think it's fine to drop this case as we have `case version if !isSupportedScalaVersion(version) =>` later anyway 👍